ex5B/ex5Spec.js

```diff
@@ -1,23 +1,23 @@
-var ex5 = require('./ex5.js');
+var firstUniqueChar = require('./ex5.js');
 
-describe("firstUniqueChar", function() {
-	it("finds another unique character in the string", function() {
-		expect(ex5.firstUniqueChar("abcdedcba")).toBe("e");
+describe("function firstUniqueChar", function() {
+	it("finds another unique character e in the string", function() {
```

Comment:
The e is irrelevant for the test name -- it doesn't matter what letter is set to find. The title of the test should not have `e` in it.